src/components/molecules/Login.tsx

```diff
@@ -0,0 +1,192 @@
+  lineHeight: 1,
+  color: indigo.indigo11,
+  boxShadow: `0 0 0 1px ${indigo.indigo7}`,
+  height: 35,
+  "&:focus": { boxShadow: `0 0 0 2px ${indigo.indigo8}` },
+  "&::placeholder": { color: mauve.mauve7 },
+});
+
+const ErrorMsg = styled("p", {
+  color: tomato.tomato8,
+  fontSize: 12,
+  marginTop: 5,
+});
+
+export default function Login({ setLoggedin }: Props) {
+  const {
+    register,
+    handleSubmit,
+    formState: { errors },
+    getValues,
+  } = useForm<LoginFormInput>();
+  const [errMessage, setErrMessage] = useState<string | null>();
+  const onSubmit: SubmitHandler<LoginFormInput> = (data) => {
+    setErrMessage("");
+
+    const formData = new FormData();
+    formData.append("username", data.email);
+    formData.append("password", data.loginPassword);
+
+    api.post("/login", formData).then((res) => {
+      Cookies.set("access_token", res.data.access_token);
+      setLoggedin(true);
+    }).catch((error: any) => {
+      Cookies.remove("access_token");
+      setErrMessage(error.response.data.detail);
+    });
+  };
+
+  return (
+    <>
+      {/* Sign In */}
+      <form onSubmit={handleSubmit(onSubmit)}>
+        <TabsContent value="tab2">
+          <Text>If you already have account, please sign in here.</Text>
+          {errMessage ? (
+            <ErrorMsg css={{ marginBottom: 10 }}>{errMessage}</ErrorMsg>
+          ) : (
+            ""
+          )}
+          <Fieldset>
+            <Label htmlFor="email">Email</Label>
+            <Input
+              type="mail"
+              id="email"
+              placeholder="sample@example.com"
+              {...register("email", {
+                required: "* This field is required",
+                pattern: {
+                  value: /^[\w\-._]+@[\w\-._]+\.[A-Za-z]+/,
+                  message: "* Invalid email address",
+                },
+              })}
+            />
+            <ErrorMsg>{errors.email?.message}</ErrorMsg>
+          </Fieldset>
+          <Fieldset>
+            <Label htmlFor="loginPassword">Password</Label>
+            <Input
+              id="loginPassword"
+              type="password"
+              {...register("loginPassword", {
+                required: "* This field is required",
+              })}
+            />
+            <ErrorMsg>{errors.loginPassword?.message}</ErrorMsg>
+          </Fieldset>
+          <Fieldset>
+            <Label htmlFor="confirmPassword">Confirm Password</Label>
+            <Input
+              id="confirmPassword"
+              type="password"
+              {...register("confirmPassword", {
+                required: "* This field is required",
+                validate: (value) => {
+                  return (
+                    value === getValues("loginPassword") ||
+                    "* Fields do not match"
+                  );
+                },
+              })}
+            />
+            <ErrorMsg>{errors.confirmPassword?.message}</ErrorMsg>
+          </Fieldset>
```

Comment:
Do you need `confirmPassword` in sign in? Also the `confirmPassword` field in `LoginFormInput` interface.
I think the `confirmPassword` should be displayed when user setting up the initial password(Sign up).